team.go

```diff
@@ -1,534 +1,561 @@
 package opslevel
 
 import (
-    "fmt"
-    "html"
-
-    "github.com/hasura/go-graphql-client"
+	"fmt"
```

Comment:
@rocktavious you might have to walk me through how you changed the formatting.  Matthew and I have both removed tab spacing in goland and it still wants to undo your indentation fixes.